vast.py

```diff
@@ -52,6 +53,64 @@
 class Object(object):
     pass
 
+def convert_cron_to_millis(schedule):
+    """Convert cron schedule to a time interval in milliseconds."""
+    try:
+        cron = croniter(schedule)  # Validate the cron schedule
+    except CroniterBadCronError:
+        raise argparse.ArgumentTypeError(f"Invalid cron schedule: '{schedule}'. "
+                                          "Make sure it follows the cron syntax format:\n"
```

Comment:
Won't this throw a error since its essentially a bunch of separate strings instead of one big one? I think you're looking for a block string with ''' abcdefg '''

Reply:
seems okay to me when I tested, seems like croniter is built to handle this